assets/js/src/form_editor/components/form_settings/selection.jsx

```diff
@@ -26,6 +26,17 @@ class Selection extends React.Component {
         .trigger('change');
     }
 
+    if (
```

Comment:
I think we need a comment or something to understand what is going on. The commit message is not very helpful here and if we come back in two years we will wonder why is this code here.